.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  root: true,
+  extends: ["@storybook/eslint-config-storybook"],
+  rules: {
+    "no-use-before-define": "off",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "eslint-comments/disable-enable-pair": "off",
+    "import/no-extraneous-dependencies": "off",
```

Comment:
I don't like turning these rules off totally.
`import/no-extraneous-dependencies`
`no-use-before-define`

For no-use-before-define, I think it makes sense to configure specifically for functions and classes.
https://eslint.org/docs/latest/rules/no-use-before-define

Reply:
I have no real opinions here. If y'all want to change it please do. I just copied this from the CLI.

Reply:
I dropped those overrides (so they are enabled again).